Python/Watermark_Images/watermark_text.py

```diff
@@ -0,0 +1,25 @@
+#Import required Image library
+from PIL import Image, ImageDraw, ImageFont
+
+#Create an Image Object from an Image
+path = '/content/sky.jpg' #main image path

```

Comment:
Is it possible to pass the image file path dynamically?

Reply:
Done.